server/lib/disaster_dashboard.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Helper functions for getting disaster dashboard data for the app config"""
+
+from google.cloud import storage
+import json
+
+EVENT_TYPE_FILE_NAME = {
```

Comment:
might just convert this into a function from CamelEvent to camel_events.json?

Reply:
done